test-fixture/src/lib.rs

```diff
@@ -392,12 +392,16 @@ impl ToString for SharedVec {
 /// Panics if the log cannot be created.
 #[must_use]
 pub fn new_neqo_qlog() -> (NeqoQlog, SharedVec) {
-    let mut trace = new_trace(Role::Client);
-    // Set reference time to 0.0 for testing.
-    trace.common_fields.as_mut().unwrap().reference_time = Some(0.0);
     let buf = SharedVec {
         buf: Arc::default(),
     };
+
+    #[cfg(feature = "bench")]
```

Comment:
Based on prior experience here, I would use this instead:

```rust
if cfg!(feature = "bench") {
    return (NeqoQlog::disabled(), buf);
}
```

Otherwise the compiler will complain unnecessarily.